lib/layers/dataUtils/geozones/configs.js

```diff
@@ -0,0 +1,282 @@
+const { rulesFns } = require('/opt/validation-rules');
+
+const rf = new rulesFns();
+
+const TIMEZONE_ENUMS = ['America/Vancouver', 'America/Edmonton', 'America/Fort_Nelson', 'America/Creston']
+
+const GEOZONE_API_PUT_CONFIG = {
```

Comment:
I hate the way I originally set up this naming convention. I think in my other configs the terminology is also PUT/UPDATE to reflect the names of the DynamoDB operations that will be employed:

PUT: create new item or completely overwrite existing item
UPDATE: update an existing item only with the provided properties and values 

To me this is now confusing because as a CRUD operation they are both 'Update'. I dunno what we are gonna do about that, just flagging it that I should probably write some more quickApiHandler documentation somewhere...